src/collection/collection.service.ts

```
@@ -1561,9 +1561,13 @@ export class CollectionService {
 	): Promise<void> {
 		try {
 			return await fetchWithLogoutJson(
-				`${getEnv('PROXY_URL')}/collections/${collectionId}/share/add-contributor?email=${
-					user.email
-				}&rights=${user.rights}`,
+				stringifyUrl({
+					url: `${getEnv('PROXY_URL')}/collections/${collectionId}/share/add-contributor`,
+					query: {
+						email: user.email || '',
```

throw if email is empty instead of sending the request with empty string